packages/eventual-send/src/E.js

```diff
@@ -304,24 +304,69 @@ export default makeE;
 
 /**
  * The awaited return type of a function.
+ * For the eventual result of an E call, @see {EResult} or @see {ECallableReturn}
  *
  * @template {(...args: any[]) => any} T
  * @typedef {T extends (...args: any[]) => infer R ? Awaited<R> : never} EReturn
  */
 
 /**
- * @template {import('./types.js').Callable} T
+ * An eventual value where remotable objects are recursively mapped to Remote types
+ *
+ * @template T
+ * @typedef {Awaited<T>} EResult
+ */
+
+/**
+ * Experimental type mapping remotable objects to Remote types
```

Comment:
In what sense is this experimental? If it is experimental, does that mean production code should not rely on it? In that case, why is it in this PR?

Reply:
It's implemented but not used, because too many things break in agoric-sdk if we try to change `EResult` to use it.

But technically one could manually use it for their own experiments, so it is defined and exported. Id' rather not lose it completely.

Reply:
Uh, ok. Reluctantly resolving. My own practice is to keep such experiments in draft PRs on the side.